sugoi-api-rest-services/src/main/java/fr/insee/sugoi/services/controller/ExportController.java

```diff
@@ -308,4 +314,28 @@ public void getAllUsers(
         groupFilter,
         response);
   }
+
+  private <T> List<String> getFieldsAsStringList(T t) {
```

Comment:
there is a problem with the rebase here

Reply:
![image](https://user-images.githubusercontent.com/75617404/183950076-cf21025a-b190-4e9f-952c-c729a8612468.png)
Shoudl I remove all this block ?